sr_description/hand/xacro/finger/distal/distal.urdf.xacro

```diff
@@ -100,6 +109,9 @@
         <child link="${prefix}${link_prefix}tip" />
         <origin xyz="0 0 0.026" rpy="0 0 0" />
       </joint>
+      <xacro:if value="${tip_sensor == 'mst'}">
```

Comment:
It would be helpful to add a comment here explaining why we have this element for MST that doesn't exist for any of the other types of sensor.

```
For mst we add a link for every taxel in the sensor
```

Reply:
good point 31dc4823371982f673fce9b2daa291a2abcae7c9